samples/tfm/tfm_psa_template/src/main.c

```diff
@@ -63,6 +67,42 @@ void dump_hex_ascii(const uint8_t *data, size_t size)
 	printk("\n");
 }
 
+
+static void print_attest_pubkey(void)
+{
+	psa_status_t status;
+	psa_key_attributes_t key_attributes = PSA_KEY_ATTRIBUTES_INIT;
+	size_t data_length;
+	uint8_t m_pub_key[ATTEST_PUBKEY_LEN];
+	psa_key_handle_t key_handle;
+
+	key_handle = mbedtls_svc_key_id_make(0, TFM_BUILTIN_KEY_ID_IAK);
+	psa_key_attributes_t attr = key_attributes;
+
+	status = psa_crypto_init();
```

Comment:
I don't think this is needed here? `psa_initial_attest_get_token()` gets called before this function, and I imagine it requires PSA Crypto to be initialized.